piccolo/engine/sqlite.py

```diff
@@ -10,6 +10,7 @@
 from decimal import Decimal
 
 from aiosqlite import Connection, Cursor, connect
+from packaging import version
```

Comment:
Where does this dependency come from? Presumably it's a requirement from another dependency. Do you think it's worth adding it explicitly to requirements.txt, in case the third party removes it as a dependency?

Reply:
it's a dependency installed by `setuptools`. I guess it wouldn't harm to add it as dependency but that'd be very odd if that was removed `setuptools` since they're for the same ogranization.